examples/asr/transcribe_speech.py

```diff
@@ -80,14 +83,15 @@ def main(cfg: TranscriptionConfig):
 
     # setup GPU
     if cfg.cuda is None:
-        cfg.cuda = torch.cuda.is_available()
-
-    if type(cfg.cuda) == int:
-        device_id = int(cfg.cuda)
+        if torch.cuda.is_available():
```

Comment:
Cfg.cuda can't be none for int, check if cfg.cuda < 0

Reply:
I'm not sure what the issue was here. cfg.cuda (now cfg.cuda_device) can be None if it is unspecified.
I think I have resolved all issues in the latest commits.